operations-stdlib/src/commonMain/kotlin/Drop.kt

```diff
@@ -0,0 +1,40 @@
+import operation.StandardLogicOperation
+import utils.asList
+import utils.secondOrNull
+import utils.thirdOrNull
+
+object Drop : StandardLogicOperation {
+    override fun evaluateLogic(expression: Any?, data: Any?): Any? =
+        with(expression.asList) {
+            val dropCandidate = firstOrNull()
+            val count = secondOrNull()
+            val mode = (thirdOrNull() as? String).toDropMode()
+
+            (count as? Int)?.let { dropCandidate.drop(it, mode) }
+        }
+
+    private fun String?.toDropMode() = when (this) {
```

Comment:
consider using sealed class

Reply:
Do you mean creating some type for unknown dropMode instead of using null?

Reply:
yep

Reply:
[not a blocker] why sealed class, not enum? In enum you could also have UNKNOWN.